### ISSUE TITLE
Load a "default" ApplicationLoader

### ISSUE THREAD

**Issue by schmitch** [playframework/playframework#6184]
### Are you looking for help?

Actually Filters could be loaded without specifing it inside the `application.conf` when you have a `Filter.class` inside the `_root_` package. The same thing would be a great addition for the `ApplicationLoader`.

**Comment by mkurz** [comment]
With modules this is possible already:

> If you call this module `Module` and place it in the root package, it will automatically be registered with Play. 

From https://www.playframework.com/documentation/2.5.x/ScalaDependencyInjection

**Comment by schmitch** [comment]
@mkurz Yes that is possible ( I also added a PR here for the Documentation https://github.com/playframework/playframework/pull/6185) still some things aren't possible without the ApplicationLoader. So it would be great to have the same thing for ApplicationLoader, than we have for Filter and Module.

**Comment by mkurz** [comment]
OK sure, I misunderstood you initial post, sorry.
